src/PointwiseFunctions/MathFunctions/Gaussian.cpp

```diff
@@ -58,6 +58,17 @@ void Gaussian::pup(PUP::er& p) {
   p | inverse_width_;
```

Comment:
Again, what are you adding the comparison operator too? (And I know you're testing it ;) )

Reply:
Why are these in a separate commit from Minkowski?

Reply:
Minkowski did not have a base class so I did it before the commit that tests serialization via base classes